### ISSUE TITLE
In prim's, mention start node in output network name

### ISSUE THREAD

**Issue by smd-faizan** [smd-faizan/CySpanningTree#5]
In prim's, mention start node in output network name

**Comment by srikanthBezawada** [comment]
How about increasing the size of the start node in the output network ?

**Comment by srikanthBezawada** [comment]
Or change the shape of the start node in the output network ?

**Comment by smd-faizan** [comment]
Good idea! lets increase the size of the Node. I am still in doubt if we should append the start node in the output network name. Whats your call?

**Comment by srikanthBezawada** [comment]
Yeah, let's rename the network as  network_startnodename. User might not be interested in changing the node properties.
